lang/hu.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+return array(
+    'validate_required'                 => '{field} mező kötelező. ',
```

Comment:
extra space at the end of the message in all messages

Reply:
Of course, thank you for review.